src/MultivariateStats.jl

```diff
@@ -427,9 +427,10 @@ $BayesianLDA_DESCR
                     within-class covariance estimator, is added to the diagonal
                     of Sw to improve numerical stability. This can be useful if
                     using the standard covariance estimator.
-* `priors=nothing`: For use in prediction with Bayes rule. if `priors = nothing` estimates the prior probabilities
-                    from the data else it uses a user specified Dictionary(`::AbstractDict{<:Union{Symbol, String}, <:Real}}`)
-                    of `class => probability` pairs for each unique class found in training data.
+* `priors=nothing`: For use in prediction with Baye's rule. If `priors = nothing` then `priors` are estimated
+                    from the class proportions in the training data. Otherwise it requires a vector containing 
+                    class probabilities specified by order of class levels ommitting class levels which are not
+                    present in training data.
 
```

Comment:
The switch to a vector sounds less safe. What's your reason for abandoning the use of a dictionary?

Reply:
I used `Vector` instead of of `Dict` for two reasons.
1. because it's much more efficient than `Dict`.
2. To avoid annotating with `Dict{Any, Float64}. (Because the class labels could be anything)
As regards the resampling issue that can be fixed.